src/test/java/com/salesforce/mirus/offsets/OffsetSerDeTest.java

```diff
@@ -47,11 +47,14 @@ public void setUp() {
     csvList =
         Arrays.asList(
             "connector-id,topic,1,123" + System.lineSeparator(),
-            "prd-logbus-source,sfdc.test.logbus__prd.ajna_test__logs.coreapp.sp1.logbus,2,345" + System.lineSeparator());
+            "prd-logbus-source,sfdc.test.logbus__prd.ajna_test__logs.coreapp.sp1.logbus,2,345"
```

Comment:
Please add some examples with "null" values to this test.

Reply:
Missed that, good catch.